components/director/internal/domain/api/repository.go

```diff
@@ -14,10 +14,16 @@ import (
 const apiDefTable string = `"public"."api_definitions"`
 
 var (
-	tenantColumn     = "tenant_id"
-	apiDefColumns    = []string{"id", "tenant_id", "bundle_id", "name", "description", "group_name", "target_url", "version_value", "version_deprecated", "version_deprecated_since", "version_for_removal"}
+	tenantColumn  = "tenant_id"
+	apiDefColumns = []string{"id", "tenant_id", "bundle_id", "package_id", "name", "description", "group_name", "target_url", "ord_id",
```

Comment:
just out of curiosity - do we need both **bundle_id** and **package_id**

Reply:
API Resource has relation both to a package and a bundle. From ORD perspective those both are two different groupings with different semantics. Package = Documentation/Visualisation grouping and Bundle = Consumption Grouping.